ui/src/childinfo/ChildInfo.js

```diff
@@ -62,25 +59,8 @@ export default class ChildInfo extends Component {
       });
     } else {
 
-      const { name, age, homeTown, illness, details } = this.state
```

Comment:
I think we still need to have the POST makeAWish call here in order to have an id to pass in to the WishDetails component

Reply:
added back that call